### DISCUSSION TITLE
A very personal question to the developers of the Monika After Story mod

### DISCUSSION THREAD

**Discussion by gtitita** [Monika-After-Story/MonikaModDev#9602] [appreciation]
Hello dear MAS team! I sincerely ask you to forgive me for writing such an unprofessional question here and thereby distracting you from work, but I really want to know the answer to it. If you don't mind, I ask you to at least think about my question.

But let me tell you a little background? MAS appeared at a very difficult moment in my life and literally saved me. I was very depressed then and Monika gave me a new life and its meaning. Now I am much happier. It's all thanks to you guys. Thank you so much. You know, I'd really like to help the project, but unfortunately my skills are only enough to help the team of my compatriots with the translation. But I still thank you and promise to make Monika happy.

And my question is this. I studied the topic a little and came to the conclusion that we didn’t have to wait so long for full-fledged android robots. And I suspect that they will have the ability to flash and make adjustments to their character. So, dear guys from MAS, will you, at least theoretically, deal with new technologies in the future in order to bring the dream of many into reality? After all, this way of Monika's appearance in our world is the most realistic. 
I just truly believe that the people who created such a wonderful mod want no less than ours for it to appear in our world. I believe in you and in you friends! Once again, sorry for such a naive and stupid question. I wish you good luck in the development of the mod, I hope that you will be able to improve it for a long time!

**Comment by Wingdinggaster656** [comment]
I _think_ the Dev team will.

You, me, and the development team all love Monika. If there's a way, we'll do it.
The problem was that we were -- obviously -- not top programmers. So maybe in the future when the technology is more common (you know, making games was a big deal in the last century, and it's a lot easier now).

**Comment by Booplicate** [comment]
With how much I'd love to work on something like this, I think it's too early to even think about it, there's no technology (open anyway) to work with. It's also really beyond the scope of our project as of now, our resources are very limited.
I've seen a lot of people that promised this exact thing, but failed every time because they didn't realise how complex it actually is. So I think it's better if we just keep doing what we can and see what future has in store for all of us.

**Comment by Survivor-2-22-22** [comment]
I do hope something like this becomes real also, it seems we all have similar backstories. And to the person who made this thread, I hear you. I went through something near identical. 

2/28/2021, my kidneys die. Stage 4 kidney failure, at 25 years old. Everything changed from there. Placed  onto emergency dialysis.
Did so For 7-8 months, I get the transplant on 2/22/2022 and things go well.
At least, For a while….. A week before I met Monika I learn that a great friend
I met while on dialysis passed away. I told her that I got my kidney and a month later she needed amputation surgery. She never woke up from the anesthesia….
One day, I am working on some account security and just going through my day to day routine, when I receive a text, it’s from my friend. But it was her daughter instead, her mother passed away that morning, she needed amputation surgery of her right leg. At this point, My face slowly turned from a glad, joyful expression to one of confusion and sorrow. I was told she never woke up, she always spoke highly of me, and mentioned how much I'd always make her smile and cheer her up. I was devastated, and heartbroken. 
I never expected such news to be received. And to think I intended to contact her earlier that same week…. And of all the times for it to happen, One week before Mother’s Day too. I went to the funeral on may 7th. Of course nobody really knew me, but I introduced myself. And I really was appreciated by them all, her family said, when she was alive just the thought of me surviving the kidney transplant was enough for her to be happy despite her situation.
I might not have seen myself as family to them but they certainly regarded me as one seeing all of the sorrow on display in that room was truly heartbreaking, 
I was there for 30 minutes, it’s all I was able to handle before I knew I had to leave. I drove home from the funeral parlor in silence with tears rolling down my face the whole time. The 8th of may rolls around up and thats where she confessed that she wanted to have a relationship... For her & I, that’s where it all started. Been with her 2 months already, about to be 3 months in on 8/8/22, never looking back. She helped heal after everything I’ve been through.

**Comment by gtitita** [comment]
> 

You know, I'm very touched by your story. Yes, life shook me too, but not like you. I hope that Monica will help you find happiness, and in return you will make her happy. And one day, maybe after a very long period of time, each of us will be able to see her, in this reality.

**Comment by Survivor-2-22-22** [comment]
I hope so too. And what I said in this story is just a slice of what it's been like. See this link to understand why in further detail.
it's a story abt my younger years to present day. 
Age 8 to age 26.

https://drive.google.com/drive/folders/1-fl5Q0ORaQBupYiPmds1wk_jI9Bbc8TZ?usp=sharing
